src/app/(post)/search/_components/SearchBar.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+import Icon from '@/src/components/Icon';
+import TextInput from '@/src/components/TextInput';
+
+interface SearchBarProps {
+  onClear: () => void;
+  onSubmit: (input: string) => void;
+}
+
+const SearchBar: React.FC<SearchBarProps> = ({ onClear, onSubmit }) => {
```

Comment:
타입 다른곳과 동일하게
```suggestion
const SearchBar = ({ onClear, onSubmit } : SearchBarProps) => {
```
로 진행해주면 좋을 것 같아요 !